_data/projects/job-requirements.yml

```
@@ -0,0 +1,8 @@
+name: Job Requirements
+desc: Organized collection of job requirements needed for successful career in tech
+site: https://github.com/crystal-dawn/job-requirements
+tags:
+- json
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/crystal-dawn/job-requirements/labels/up%20for%20grabs*
```

The trailing `*` here causes the URL to be incorrect:

```suggestion
  link: https://github.com/crystal-dawn/job-requirements/labels/up%20for%20grabs
```

---

Thank you!